rest_api/rest_api/pipeline/__init__.py

```diff
@@ -25,12 +32,70 @@ def setup_pipelines() -> Dict[str, Any]:
     pipelines = {}
 
     # Load query pipeline
-    query_pipeline = Pipeline.load_from_yaml(Path(config.PIPELINE_YAML_PATH), pipeline_name=config.QUERY_PIPELINE_NAME)
+    # query_pipeline = Pipeline.load_from_yaml(Path(config.PIPELINE_YAML_PATH), pipeline_name=config.QUERY_PIPELINE_NAME)
+
+    # ------------------
+    document_store = ElasticsearchDocumentStore(host="host.docker.internal")
+    extractive_document_store = ElasticsearchDocumentStore(
+        host="host.docker.internal", index="rulebook", embedding_dim=768
+    )
+    faq_document_store = ElasticsearchDocumentStore(
+        host="host.docker.internal", index="faq", embedding_dim=384, similarity="cosine"
+    )
+
+    extractive_reader_option = "deepset/roberta-base-squad2"
+    faq_retriever_option = "sentence-transformers/all-MiniLM-L6-v2"
+
+    faq_retriever = EmbeddingRetriever(
+        document_store=faq_document_store,
+        embedding_model=faq_retriever_option,
+        use_gpu=False,
```

Comment:
```suggestion
        use_gpu=torch.cuda.is_available(),
```
Hey Rafal, would it be better to check for GPU usage to speed up the update_embeddings call on the faq_document_store?

Reply:
Sure thing!